README.md

```diff
@@ -47,31 +56,90 @@ and ask!
-all parts of the system interact, but as a community we can lean on each other
-to build the right design and approach before getting too deep into an
-implementation.
-
-## When to Comment on an Enhancement Issue
-
-Please comment on the enhancement issue to:
-- request a review or clarification on the process
-- update status of the enhancement effort
-- link to relevant issues in other repos
-
-Please do not comment on the enhancement issue to:
-- discuss a detail of the design, code or docs. Use a linked-to-issue or design PR
-  for that
+## How are Enhancements Reviewed and Approved?
+
+The author of an enhancement is responsible for managing it through
+the review and approval process, including soliciting feedback on the pull request
+and in meetings, if necessary.
+
+Each enhancement should have at least one "approver" and several
+reviewers designated in the header of the document.
+
+The approver assists authors who may not be familiar with the process,
+the project, or the maintainers. They may provide advice about who
+should review a specific proposal and point out deadlines or other
+time-based criteria for completing work. The approver is responsible
+for recognizing when consensus among reviewers has been reached so that a proposal is
+ready to be approved, or formally rejected. In cases where consensus
+is not emerging on its own, the approver may also step in as a
+mediator. The approver does not need to be a subject-matter expert for
+the subject of the design, although it can help if they are.
+
+Choosing the appropriate approver depends on the scope of an
+enhancement. If it is limited in scope to a given team or component,
+then a peer or lead on that team or pillar is appropriate.  If an
+enhancement captures something more broad in scope, then a member of
+the OpenShift staff engineers team or someone they delegate would be
+appropriate.  Examples of broad scope are proposals that change the
+definition of OpenShift in some way, add a new required dependency, or
+change the way customers are supported.  Use your best judgement to
+determine the level of approval needed.  If you’re not sure, ask a
+staff engineer to help find a good approver by posting in
+`#forum-arch` on the CoreOS Slack server and tagging
+`@aos-staff-engineers`.
+
+The set of reviewers for an enhancement proposal can be anyone that
+has an interest in this work or the expertise to provide a useful
+input/assessment.  At a minimum, the reviewers must include a
+representative of any team that will need to do work for this EP, or
+whose team will own/support the resulting implementation. Be mindful
+of the workload of reviewers, however, and the challenge of finding
+consensus as the group of reviewers grows larger. Clearly indicating
+what aspect of the EP you expect each reviewer to be concerned with
+will allow them to focus their reviews.
+
+## How Can an Author Help Speed Up the Review Process?
```

Comment:
@dhellmann  @bparees sorry for being late for the review.

I would suggest to add a task in the epic to follow up and allow tracking the progress of the enhancement.
If it requires significant effort from other teams (review as well), it should be added as a task on their board as well.
Then, the weekly automation can aggregate enhancement progress (since we include a link to the jira in the enhancement as well), with jira status, including the fixVersion of that epic, to mark and alert on critical enhancements that are not moving forward.

Reply:
+1 to ensuring EP work is tracked in jira

Reply:
(but i think it's already supposed to be at least linked?)

Reply:
See https://github.com/openshift/enhancements/pull/1087